test/onnx/test_fx_op_consistency.py

```diff
@@ -32,13 +32,16 @@
 import parameterized
 
 import torch
+from test_fx_to_onnx_with_onnxruntime import (
+    _run_test_with_fx_to_onnx_exporter_and_onnx_runtime,
```

Comment:
Avoid importing from a test file. Prefer to extract this function out to a common file and mark as non private?

Reply:
We should avoid importing private functions.

Reply:
I willl update it in https://github.com/pytorch/pytorch/pull/99434